src/main/resources/db/migration/v0.34.0.3__gene_expression_annotation.sql

```diff
@@ -0,0 +1,73 @@
+CREATE SEQUENCE public.expressionannotation_seq         START WITH 1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE TABLE expressionannotation (
+	id bigint CONSTRAINT expressionannotation_pkey PRIMARY KEY,
+	relation_id bigint,
+	whenexpressedstagename VARCHAR(2000),
+	whereexpressedstatement VARCHAR(2000)
+);
+
+ALTER TABLE expressionannotation
+	ADD CONSTRAINT relation_id_fk
+		FOREIGN KEY (relation_id) REFERENCES vocabularyterm (id);
+
+CREATE INDEX expressionannotation_whenexpressedstagename ON expressionannotation USING btree (whenexpressedstagename);
```

Comment:
Our convention for index names is `<table_name>_<field_name>_index`, so `expressionannotation_whenexpressedstagename_index` in this case

Reply:
Also, I don't see these indexes defined in your ExpressionAnnotation entity class

Reply:
Fixed!